src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -332,6 +333,26 @@
     <TrimmerRootDescriptor Include="$(MSBuildThisFileDirectory)ILLink.Descriptors.xml" />
   </ItemGroup>
 
+  <!--
+      These tests are currently incompatible with TestReadyToRun.
+      We have a tracking issue herehttps://github.com/dotnet/runtime/issues/94189
+  -->
+  <ItemGroup Condition="'$(TestReadyToRun)' == 'true'">
+    <Compile Remove="System\Type\TypeTests.Get.cs" />
+    <Compile Remove="System\Runtime\JitInfoTests.cs" />
+    <Compile Remove="System\EnumTests.cs" />
+  </ItemGroup>
+
+  <!--
+      Timezone tests are not working on Linux on CI when TestReadyToRun is enabled.
+      It might be something missing with the machines. We have a tracking issue here:
+      https://github.com/dotnet/runtime/issues/94190
+  -->
+  <ItemGroup Condition="'$(TargetOS)' == 'linux' and '$(TestReadyToRun)' == 'true'">
+    <Compile Remove="System\TimeZoneInfoTests.cs" />
+    <Compile Remove="$(CommonTestPath)System\TimeProviderTests.cs" />
+  </ItemGroup>
```

Comment:
This question still remains for me. Why do we remove these tests at compile item instead of conditioning them out at run time? That's unusual.

Reply:
Sorry Viktor, it was the only way I knew of excluding tests. I will remove these item groups now that Michal showed me how we exclude libraries tests.